openapi/nexus.json

```diff
@@ -3436,6 +3436,69 @@
         }
       }
     },
+    "/v1/system/hardware/disks/{vendor}/{serial}/{model}": {
```

Comment:
A series of open identifiers can create issues, e.g., you can't add `/vendor/specific-page`. We generally try to avoid it.

Reply:
Is there a reason why we can't (or shouldn't) use the UUID associated with the DB entry?

Reply:
I would like to, but I need a variant of the `LookupPath` that can lookup by **either** UUID or (vendor, serial, model). We currently look up the disk by (vendor, serial, model) from the Sled Agent's calls to the internal API.

Reply:
Gave this a shot -- it requires an intermediate DB operation to translate from UUID -> (vendor, serial, model), which could probably be avoided if we could use the string tuple as a name.

Reply:
I think that's much better from an API POV, thanks.